webpack.config.js

```diff
@@ -1,42 +1,57 @@
 const path = require('path');
 
+const HtmlWebPackPlugin = require("html-webpack-plugin");
```

Comment:
`html-webpack-plugin` is missing from `package.json`, and this doesn't work on my machine unless I add that dependency manually. Can you include it in the PR?

Reply:
Package 'html-webpack-plugin' imported into package.json